Date: Oct 31 08:24:49 2017

From: e9c25caa87cc5772aad1e254c46fda25047e1a38 Oct 31 08:24:49 2017

Subject: [PATCH] RDK-20604 : Fixing GCC_6.4 Wno-narrowing compilation error

	With the latest GCC_6.4 version some of the builds are failing
	in opencdm module for wnarrowing error. Hence, used the patch.

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: nhanasi <Naveenkumar_Hanasi@comcast.com>
---

diff --git a/src/browser/wpe/test/Makefile b/src/browser/wpe/test/Makefile
index d63ea05..1a56566 100644
--- a/src/browser/wpe/test/Makefile
+++ b/src/browser/wpe/test/Makefile
@@ -40,7 +40,7 @@ OCDM_INCLUDES = -I $(OCDM_ROOT_PATH) \
                 -I $(OCDM_WPE_ROOT_PATH)/opencdm \
                 $(OCDM_HEADER_PATH)
 
-CXFLAGS := -std=c++11 -g $(OCDM_INCLUDES) -pthread $(CXXFLAGS)
+CXFLAGS := -std=c++11 -g -Wno-narrowing $(OCDM_INCLUDES) -pthread $(CXXFLAGS)
 
 all: ocdm_client ocdm_decryptor
 	@echo "Building ocdm client"
